src/Microsoft.Health.Dicom.Core.UnitTests/Features/CustomTag/CustomTagEntryFormalizerTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Dicom;
+using Microsoft.Health.Dicom.Core.Extensions;
+using Microsoft.Health.Dicom.Core.Features.Common;
+using Microsoft.Health.Dicom.Core.Features.CustomTag;
+using Microsoft.Health.Dicom.Core.UnitTests.Features.CustomTag;
+using Xunit;
+
+namespace Microsoft.Health.Dicom.Core.UnitTests.Features.ChangeFeed
+{
+    public class CustomTagEntryFormalizerTests
+    {
+        private ICustomTagEntryFormalizer _customTagEntryFormalizer;
+
+        public CustomTagEntryFormalizerTests()
+        {
+            _customTagEntryFormalizer = new CustomTagEntryFormalizer(new DicomTagParser());
```

Comment:
>_customTagEntryFormalizer = new CustomTagEntryFormalizer(new DicomTagParser()); [](start = 12, length = 79)

This isn't stateful, right? So we can probably just initialize this as a `static` `readonly` `private` member